core/src/App.html

```diff
@@ -577,7 +577,21 @@
       handleNavClick(node) {
         this.getUnsavedChangesModalPromise().then(() => {
           this.closeLeftNav();
-          Routing.handleRouteClick(node, this);
+          if (node.openNodeInModal) {
+            const route = RoutingHelpers.buildRoute(
+              node,
+              `/${node.pathSegment}`
+            );
+            if (node.openNodeInModal)
+              this.openViewInModal(
+                route,
+                typeof node.openNodeInModal === 'boolean'
```

Comment:
Yes because it is cool. Also Pete liked it.. and it is documented. See docu..
`Optionally you can set the **openNodeInModal** property to true to use default title and size.`

Reply:
Ok, I missed that part in the docu